sys-meta/cpio/cpio-0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="CPIO symlink"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="+gnu libarchive"
+REQUIRED_USE="^^ ( gnu libarchive )"
+
+RDEPEND="
+	gnu? ( >=app-arch/cpio-2.13-r4 )
+	libarchive? ( app-arch/libarchive )
+	!<app-arch/cpio-2.13-r4
+"
+
+src_install() {
+	if use gnu; then
+		dosym gcpio /bin/cpio
+		newman - cpio.1 <<-EOF
+			.so gcpio.1
+		EOF
```

Comment:
Just style, but for one short line I'd do:
```bash
newman - cpio.1 <<<".so gpcio.1"
```
That aside, I like the sys-meta idea at a glance, especially for simple things you pick once and probably never switch again.